docs/troubleshooting.rst

```diff
@@ -48,3 +48,21 @@ or file an issue on `GitHub
 <https://github.com/kanisterio/kanister/issues>`_. A `mailing list
 <https://groups.google.com/forum/#!forum/kanisterio>`_ is also
 available if needed.
+
+Validating webhook for Blueprints
+=================================
+For the validating webhook to work, the Kubernetes API Server needs to
```

Comment:
It will be helpful to describe the symptoms of an unreachable webhook. E.g., what is the output of `kubectl apply -f blueprint.yaml`?